pyServer/manifests/linux/monitor-mgmt

```diff
@@ -1,39 +1,49 @@
-echo,#Microsoft Monitoring Agent for Linux Log File
-copy,/var/opt/microsoft/omsagent/*/log/omsagent.log
+echo,# OMS Agent for Linux Log File
+copy,/var/opt/microsoft/omsagent/*/log/omsagent*
```

Comment:
syntax looks good.  Just a general wildcard note that this approach can result in a large number of collected files if non-circular logging is used

Reply:
We have a logrotate policy in place for omsagent.log :)